pkg/resources/templates/config/nifi_properties.go

```diff
@@ -217,11 +309,26 @@ nifi.cluster.firewall.file=
 nifi.cluster.flow.election.max.wait.time=1 mins
 nifi.cluster.flow.election.max.candidates=
 
+# cluster load balancing properties #
+nifi.cluster.load.balance.host=
+nifi.cluster.load.balance.port=6342
```

Comment:
I think we need to move this definition here: https://github.com/konpyutaika/nifikop/blob/master/pkg/resources/templates/config/nifi_properties.go#L273

If I understand the property correctly

Reply:
Did the property names change or are these different? One property is `nifi.cluster.load.balance.port` and the other is `nifi.cluster.node.load.balance.port`

Reply:
Strangely, I don't find any mention of `nifi.cluster.node.load.balance.port` in the code of NiFi. I went back to `1.9.1`. Only found it in the documentation. But `nifi.cluster.load.balance.port` does exist, even in `1.9.1`. Maybe our thing does nothing 😄.

Reply:
![image](https://github.com/konpyutaika/nifikop/assets/31806759/b0e719de-97bd-48f2-8e6e-9ddf3bce723e)

I guess I can correct the property.

Reply:
Property corrected.